test-app/spec/system/viewing_all_job_postings_spec.rb

```diff
@@ -25,10 +25,10 @@
     posting = create :job_posting
     visit job_postings_path
 
-    expect(has_job_posting?(posting)).to be true
+    expect(job_posting?(posting)).to be true
   end
 
-  def has_job_posting?(posting)
+  def job_posting?(posting)
     within(".job-posting-#{posting.id}") do
       expect(page).to have_css("td.posting-title", text: posting.title)
     end
```

Comment:
Since this is kind of acting as a matcher (except not really), the older syntax seems preferable. Since this is the only location in this spec that seems to be using the helper, another option would be to just remove it, and move this block right into the main body of the spec:
```
 within(".job-posting-#{posting.id}") do
      expect(page).to have_css("td.posting-title", text: posting.title)
    end
```

No need for a helper if we're not going to re-use it, right?

Reply:
This was also Rubocop, so I don't have strong feeling about this.  If it's only used in one place I'll go ahead and remove the function.